simulation_rate: 100.0 # Hz
clock_rate: 100.0 # Hz
realtime_factor: 1.0 # [-]

g: 9.81 # [ms^-2]

iterate_without_input: true

input_timeout: 1.0 # [s]

sensors:
  lidar:
    enabled: true
    rate: 10.0 # [Hz]
  lidar_seg:
    enabled: false
    rate: 10.0 # [Hz]
  rgb:
    enabled: true
    rate: 10.0 # [Hz]
  depth:
    enabled: false
    rate: 10.0 # [Hz]
  seg:
    enabled: true
    rate: 5.0 # [Hz]
  color_depth:
    enabled: true
    rate: 5.0 # [Hz]

frames:
  world:
    name: "simulator_origin"
    prefix_with_uav_name: false
  fcu:
    name: "fcu"
    publish_tf: false
  rangefinder:
    name: "garmin"
    publish_tf: true

# randomization of the initial conditions around the desired spawn point
randomization:

  enabled: false

  bounds:
    x: 15.0
    y: 15.0
    z: 15.0

ground:
  enabled: true
  z: 0.0 # [m]

# * when enabled, this will place a temporary patch of ground just under the spawn location
# * this patch will disappear after the total_thrust > 0.9*hover_thrust
individual_takeoff_platform:
  enabled: false
